plugins/catalog/src/index.ts

```diff
@@ -20,6 +20,6 @@ export * from './api/types';
 export * from './routes';
 export { useEntityCompoundName } from './components/useEntityCompoundName';
 export { Router } from './components/Router';
-export { useEntity } from './hooks/useEntity';
+export { useEntity, EntityContext } from './hooks/useEntity';
```

Comment:
I should probably highlight this as it's likely unexpected. I found I needed to export this to test a plugin component which uses useEntity().